index.html

```diff
@@ -5,8 +5,150 @@
     <title>HTML Academy: Седона</title>
   </head>
   <body>
+  <header class="page-header">
+    <nav class="navigation">
+      <a href="http://htmlacademy.ru"><img class="logo" src="" alt="Логотип Город Седон."></a>
+      <ul class="navigation-list">
+        <li class="navigation-item">
+          <a class="navigation-link" href="index.html">Главная</a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="about.html">О Седоне</a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="hotels.html">Гостиницы</a>
+        </li></ul>
+        <ul class="navigation-user">
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden">Поиск</span>
+            </a>
+          </li>
+          <li class="navigation-user">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden">Нравится</span>
+            </a>
+          </li>
+          <li class="navigation-user">
+            <a class="navigation-link" href="#">
+              <p>Хочу сюда</p>
+            </a>
+          </li></ul>
+    </nav>
+</header>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+<main class="main-index">
+<h1 class="visually-hidden">Седона</h1>
+<img class="logo" src="" alt="Название Седона.">
+
+<section>
+  <h2>Седона — небольшой городок в Аризоне,
+    заслуживающий большего!</h2>
+    <p>Рассмотрим причины, по которым Седона круче, чем Гранд-Каньон!</p>
+</section>
+
+  <ul class="advantages-list">
+    <li class="advantages-item">
+    <h3 class="advantages-title">Настоящий городок</h3>
+    <p class="advantages-description">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
+    <img class="picture" src="" alt="Изображение Седоны с кустами на переднем фоне и флагами">
+  < /li>
```

Comment:
```suggestion
  </li>
```
То есть лишний пробел после символа `<`.